scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixFileEvaluation.java

```diff
@@ -22,6 +22,8 @@
      */
     boolean isSuccessful();
 
+    Optional<String> getMessageError();
```

Comment:
could you add some javadoc? does "error" include lint errors?

Reply:
ok. It includes lintErrors.

Reply:
sorry if I wasn't clear but I was talking about `getMessageError` - I think the `getErrors` are clear enough given their signature so the javadoc there is redundant.